ProcessLib/Process.h

```diff
@@ -57,7 +57,10 @@ class Process
                      const double delta_t, const int process_id);
 
     /// Postprocessing after a complete timestep.
-    void postTimestep(GlobalVector const& x);
+    void postTimestep(GlobalVector const& x, int const processs_id);
+
+    void postNonLinearSolver(GlobalVector const& x, const double t,
```

Comment:
Docu.
Just for understanding, why is this function now necessary, and it was not required before?

Reply:
This  corrects stress and strain for output and for coupling. In the present master branch, the strain together with stress are calculated by using the displacement of one iteration step forward  before nonlinear solver convergences.